tests/blackbox/routes/items/m2a.test.ts

```diff
@@ -2170,6 +2170,8 @@ describe.each(common.PRIMARY_KEY_TYPES)('/items', (pkType) => {
 
 			describe('updates the auto increment value correctly', () => {
 				it.each(vendors)('%s', async (vendor) => {
+					if (vendor !== 'postgres') return;
```

Comment:
I think we update this condition across all tests to run for all vendors except the specifically excluded ones.

Reply:
alright!